virttest/utils_net.py

```diff
@@ -3748,23 +3764,27 @@ def get_host_iface():
     return [_.strip() for _ in re.findall("(.*):", host_iface_info)]
 
 
-def get_host_default_gateway(iface_name=False):
+def get_default_gateway(iface_name=False, *session):
```

Comment:
why do we consider list `*session` here ?

Reply:
When we need to get guest's default gateway, it need to pass the session as a function argument. When we need to get host's default gateway, this argument is not needed.

Reply:
agree, shouldn't it be `session=None` ? currently it declares it as a list which is not required.

Reply:
Great, Thank you very much! 
I have updated it.
And the related PRs in tp-libvirt and tp-qemu:
tp-libvirt: https://github.com/autotest/tp-libvirt/pull/2380
tp-qemu: https://github.com/autotest/tp-qemu/pull/1962